docs/source/models.rst

```diff
@@ -486,8 +487,9 @@ Faster R-CNN ResNet-50 FPN              0.2288               0.0590
 Faster R-CNN MobileNetV3-Large FPN      0.1020               0.0415              1.0
 Faster R-CNN MobileNetV3-Large 320 FPN  0.0978               0.0376              0.6
 RetinaNet ResNet-50 FPN                 0.2514               0.0939              4.1
-SSD VGG16                               0.2093               0.0744              1.5
-SSDlite MobileNetV3-Large               0.1773               0.0906              1.5
+SSD300 VGG16                            0.2093               0.0744              1.5
+SSD512 ResNet-50                        0.2316               0.0772              3.0
```

Comment:
For the future: we need to change those tables as they are misleading for now -- the `test time` column for the SSD models is for a batch size of 4 per GPU, while for Faster R-CNN it was for a batch size 2.

Reply:
Maybe do a back of the envelop estimation to bring them to comparable batch-sizes?